views/forms/ControlledMultiSelect/index.jsx

```diff
@@ -21,6 +21,15 @@ const ControlledMultiSelect = (props) => {
               aria-invalid={!!fieldState.error ? 'true' : 'false'}
               label={props.label}
               helperText={fieldState.error?.message}
+              InputProps={{
+                ...params.InputProps,
+                startAdornment: (
+                  <>
+                    {props.startAdornment}
+                    {params.InputProps.startAdornment}
+                  </>
```

Comment:
Should these be wrapped in the InputAdornment..?

Reply:
this shouldn't be, but the prop should be. If we wrap this in an `InputAdornment` then the selected options don't show up in the text field.